components/IconCorrect/IconCorrect.stories.tsx

```diff
@@ -0,0 +1,18 @@
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+
+import { IconCorrect } from "./IconCorrect";
+
+export default {
+  title: "Icon Correct",
```

Comment:
Acá se puede usar "grouping" https://storybook.js.org/docs/react/writing-stories/naming-components-and-hierarchy#grouping
En ese sentido, el título puede ser "Iconos/Checkmark"